src/test/python/deep_qa_test/models/reading_comprehension/attention_sum_reader_test.py

```diff
@@ -35,5 +37,13 @@ def test_train_does_not_crash(self):
                 },
                 "embedding_size": 5,
         }
-        solver = get_model(AttentionSumReader, args)
-        solver.train()
+        model = get_model(AttentionSumReader, args)
+        model.train()
+
+        # load the model that we serialized
+        loaded_model = get_model(AttentionSumReader, args)
+        loaded_model.load_model()
```

Comment:
I think there are some things that definitely won't work right now, but at least if you just want to load and predict, things should work.  That's good to know.